src/network/networkd-network.h

```diff
@@ -78,11 +78,13 @@ struct Network {
         LIST_HEAD(Condition, conditions);
 
         /* Master or stacked netdevs */
+        NetDev *batadv;
```

Comment:
Still this seems not used.

Reply:
When I remove this and run a ninja -C build the following happens:
```
[37/83] Compiling C object src/network/libnetworkd-core.a.p/networkd-link.c.o
FAILED: src/network/libnetworkd-core.a.p/networkd-link.c.o
cc -Isrc/network/libnetworkd-core.a.p -Isrc/network -I../src/network -Isrc/basic -I../src/basic -Isrc/boot -I../src/boot -Isrc/home -I../src/home -Isrc/shared -I../src/shared -Isrc/systemd -I../src/systemd -Isrc/journal -I../src/journal -Isrc/journal-remote -I../src/journal-remote -Isrc/nspawn -I../src/nspawn -Isrc/resolve -I../src/resolve -Isrc/timesync -I../src/timesync -I../src/time-wait-sync -Isrc/login -I../src/login -Isrc/udev -I../src/udev -Isrc/libudev -I../src/libudev -Isrc/core -I../src/core -Isrc/shutdown -I../src/shutdown -I../src/xdg-autostart-generator -I../src/libsystemd/sd-bus -I../src/libsystemd/sd-device -I../src/libsystemd/sd-event -I../src/libsystemd/sd-hwdb -I../src/libsystemd/sd-id128 -I../src/libsystemd/sd-netlink -I../src/libsystemd/sd-network -I../src/libsystemd/sd-resolve -Isrc/libsystemd-network -I../src/libsystemd-network -I. -I.. -I../src/network/netdev -I../src/network/tc -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=gnu99 -g -Wno-unused-parameter -Wno-missing-field-initializers -Wno-unused-result -Wno-format-signedness -Werror=undef -Wlogical-op -Wmissing-include-dirs -Wold-style-definition -Wpointer-arith -Winit-self -Wfloat-equal -Wsuggest-attribute=noreturn -Werror=missing-prototypes -Werror=implicit-function-declaration -Werror=missing-declarations -Werror=return-type -Werror=incompatible-pointer-types -Werror=format=2 -Wstrict-prototypes -Wredundant-decls -Wmissing-noreturn -Wimplicit-fallthrough=5 -Wshadow -Wendif-labels -Wstrict-aliasing=2 -Wwrite-strings -Werror=overflow -Werror=shift-count-overflow -Werror=shift-overflow=2 -Wdate-time -Wnested-externs -Wno-maybe-uninitialized -ffast-math -fno-common -fdiagnostics-show-option -fno-strict-aliasing -fvisibility=hidden -fstack-protector -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=shadow -include config.h -fPIC -MD -MQ src/network/libnetworkd-core.a.p/networkd-link.c.o -MF src/network/libnetworkd-core.a.p/networkd-link.c.o.d -o src/network/libnetworkd-core.a.p/networkd-link.c.o -c ../src/network/networkd-link.c
../src/network/networkd-link.c: In function ‘link_update_operstate’:
../src/network/networkd-link.c:270:65: error: ‘Network’ {aka ‘struct Network’} has no member named ‘batadv’
  270 |                 link_update_master_operstate(link, link->network->batadv);
      |                                                                 ^~
../src/network/networkd-link.c: In function ‘link_set_nomaster’:
../src/network/networkd-link.c:971:26: error: ‘Network’ {aka ‘struct Network’} has no member named ‘batadv’
  971 |         if (link->network->batadv || link->network->bridge || link->network->bond || link->network->vrf)
      |                          ^~
../src/network/networkd-link.c: In function ‘link_drop’:
../src/network/networkd-link.c:1797:58: error: ‘Network’ {aka ‘struct Network’} has no member named ‘batadv’
 1797 |                 link_drop_from_master(link, link->network->batadv);
      |                                                          ^~
../src/network/networkd-link.c: In function ‘link_enter_join_netdev’:
../src/network/networkd-link.c:1931:26: error: ‘Network’ {aka ‘struct Network’} has no member named ‘batadv’
 1931 |         if (link->network->batadv) {
      |                          ^~
In file included from ../src/basic/macro.h:657,
                 from ../src/basic/alloc-util.h:9,
                 from ../src/network/networkd-link.c:9:
../src/network/networkd-link.c:1934:62: error: ‘Network’ {aka ‘struct Network’} has no member named ‘batadv’
 1934 |                            LOG_NETDEV_INTERFACE(link->network->batadv),
      |                                                              ^~
../src/basic/log.h:269:70: note: in definition of macro ‘log_struct_errno’
  269 |                             error, PROJECT_FILE, __LINE__, __func__, __VA_ARGS__, NULL)
      |                                                                      ^~~~~~~~~~~
../src/network/networkd-link.c:1932:17: note: in expansion of macro ‘log_struct’
 1932 |                 log_struct(LOG_DEBUG,
      |                 ^~~~~~~~~~
../src/network/networkd-link.c:1934:28: note: in expansion of macro ‘LOG_NETDEV_INTERFACE’
 1934 |                            LOG_NETDEV_INTERFACE(link->network->batadv),

```